packages/teleport/src/Discover/Shared/SelectCreatable/SelectCreatable.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2022 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import ReactSelectCreatable from 'react-select/creatable';
```

Comment:
Maybe we should update react-select from 3.0.8 to latest 5.5.0. It has build in typescript support. We could also do that in separate task.
Alternatively we can install @types/react-select for now ensure we are using SelectCreatable in correct way.

Reply:
Next week I'll revive my "type the design system" PR that does this

Reply:
i tried installing the type and ran into a bunch of typescript errors unrelated to my changes so I just left things as is (i'm sorry @ryanclark it will be a headache, or I think I should be the one upgrading it, might have the most context but i don't care either way)

Reply:
tracking in this issue, b/c i found other issues https://github.com/gravitational/teleport/issues/17553